php_phongo.c

```diff
@@ -2417,7 +2414,8 @@ static const zend_function_entry mongodb_functions[] = {
 	ZEND_NS_NAMED_FE("MongoDB\\BSON", fromPHP, PHP_FN(MongoDB_BSON_fromPHP), ai_bson_fromPHP)
 	ZEND_NS_NAMED_FE("MongoDB\\BSON", toPHP, PHP_FN(MongoDB_BSON_toPHP), ai_bson_toPHP)
 	ZEND_NS_NAMED_FE("MongoDB\\BSON", toJSON, PHP_FN(MongoDB_BSON_toJSON), ai_bson_toJSON)
-	ZEND_NS_NAMED_FE("MongoDB\\BSON", toExtendedJSON, PHP_FN(MongoDB_BSON_toExtendedJSON), ai_bson_toExtendedJSON)
+	ZEND_NS_NAMED_FE("MongoDB\\BSON", toCanonicalJSON, PHP_FN(MongoDB_BSON_toCanonicalJSON), ai_bson_toJSON)
+	ZEND_NS_NAMED_FE("MongoDB\\BSON", toRelaxedJSON, PHP_FN(MongoDB_BSON_toRelaxedJSON), ai_bson_toJSON)
```

Comment:
Correct. The parser in libbson has logic to read all three formats. This was actually something I fixed in [CDRIVER-2223](https://jira.mongodb.org/browse/CDRIVER-2223).